### PR TITLE
Add changesets to React Live

### PR DESCRIPTION
**Pull request by carloskelly13**
- [x] Added token to GitHub Actions
- [x] Added configuration for Changesets (this PR)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,10 @@
     "install:docs": "yarn --cwd website install",
     "start:docs": "yarn --cwd website start",
     "build:docs": "yarn --cwd website build",
-    "format:docs": "prettier --write docs"
+    "format:docs": "prettier --write docs",
+    "prepare": "yarn build",
+    "changeset": "changeset",
+    "version": "yarn changeset version && yarn install --frozen-lockfile"
```
Comment (ryan-roemer) [inline package.json:26]: I forget the most recent reference, but isn't the `yarn install` _meant_ to update the lockfile due to changes after `changeset version` (and thus don't freeze it).

File: .changeset/config.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+  "changelog": [
+    "@svitejs/changesets-changelog-github-compact",
```
Comment (ryan-roemer) [inline .changeset/config.json:4]: I don't see the dependency for this?

File: package.json (hunk 1 above)
Comment (carloskelly13) [reply package.json:25]: It's meant to fail if the lockfile changes during install which is our typical command we use in our version task.

Comment (ryan-roemer) [reply package.json:25]: See, e.g.: https://github.com/FormidableLabs/victory/blob/main/package.json#L118

```js
"version": "pnpm changeset version && pnpm install --no-frozen-lockfile",
```

https://github.com/FormidableLabs/victory/pull/2398

**Review by gksander — APPROVED** [review]
Sick

**Pull request merged**